test/com/facebook/buck/apple/xcode/ProjectIntegrationTest.java

```diff
@@ -182,6 +182,23 @@ public void generatingCombinedProject() throws IOException {
   }
 
   @Test
+  public void generatingRootDirectoryProject() throws IOException {
+    ProjectWorkspace workspace = TestDataHelper.createProjectWorkspaceForScenario(
+        this,
+        "generating_root_directory_project",
+        temporaryFolder);
+    workspace.setUp();
+
+    ProjectWorkspace.ProcessResult result = workspace.runBuckCommand(
+        "project",
+        "//:bundle");
+    result.assertSuccess();
+
+    workspace.verify();
```

Comment:
This isn't doing anything: you don't have any `.expected` files in your test data.